tests/unit/src/test/scala/tests/TreeViewLspSuite.scala

```diff
@@ -1,9 +1,50 @@
 package tests
 
+import scala.collection.SortedSet
 import scala.meta.internal.tvp.TreeViewProvider
 
 object TreeViewLspSuite extends BaseLspSuite("tree-view") {
 
+  /** The libraries we expect to find for tests in this file.
+   *
+   * @note this value changes depending on the JVM version in use as some JAR
+   *       files have moved to become modules on JVM > 8.
+   */
+  val expectedLibraries: SortedSet[String] = {
```

Comment:
Thank you for fixing this! 👏

The biggest reason for delaying the upgrade to jdk 11 in our ci was that tests needed to be updated.